src/overview/reducer.js

```diff
@@ -28,6 +28,15 @@ const defaultState = {
     },
     showFilter: false,
     showOnlyBookmarks: false,
+    pageIdForTag: '',
+    newTag: '',
+    resultTags: [],
+    suggestedTags: [],
+    deleteTags: [],
+    hoveredTagResult: '',
+    tagSearchValue: '',
+    tags: [],
+    tagExpandedPageId: '',
```

Comment:
I don't understand the purpose of this state either. Can you explain to me what it's used for?